sys-cluster/rdma-core/rdma-core-9999.ebuild

```diff
@@ -33,7 +33,7 @@ COMMON_DEPEND="
 DEPEND="${COMMON_DEPEND}
 	python? (
 		$(python_gen_cond_dep '
-			dev-python/cython[${PYTHON_USEDEP}]
+			dev-python/cython[${PYTHON_MULTI_USEDEP}]
```

Comment:
That’s the obsolete one :)

Reply:
Oops, merged after an ack from zlogene on IRC... Will fix after the fact.